package.json

```diff
@@ -1281,6 +1292,49 @@
           "type": "integer",
           "default": 1,
           "description": "Number of header rows to assume. Set to 0 to disable.`"
+        },
+        "latex-workshop.formattedPaste.image.method": {
+          "type": "string",
+          "enum": [
+            "leave",
+            "copy",
+            "move"
+          ],
+          "enumDescriptions": [
+            "reference the image where it is",
+            "copy image to target location",
+            "move image to target location"
+          ],
+          "default": "leave",
+          "description": "When pasting the path to an image, determine how it is treated. With copy and move 'target location' is graphicsdir or cwd"
+        },
+        "latex-workshop.formattedPaste.image.template": {
```

Comment:
Are there other possible templates than this one?

Reply:
Yes. For instance what I will probably set that to is:
```json
        [
            "\\begin{figure}[!htb]",
            "\t\\centering",
            "\t\\includegraphics[]{${imageFilePath}}",
            "\t\\caption{${imageFilePathWithoutExt}}",
            "\t\\label{fig:${imageFilePathWithoutExt}}",
            "\\end{figure}"
        ]
```

Reply:
I'll mark this resolved for now but let me know if you want me to do anything with this.